web/download_manager.js

```diff
@@ -86,20 +86,20 @@ if (typeof PDFJSDev === 'undefined' || PDFJSDev.test('GENERIC || CHROME')) {
     },
 
     download: function DownloadManager_download(blob, url, filename) {
-      if (!URL) {
```

Comment:
> I just tested it out on IE11. This change makes IE11 use the downloadUrl fallback rather than navigator.msSaveBlob.

Hmm, in that case I suppose that it's probably better to move the IE block to the top of `download` method after all. Sorry about misleading you here!
```JavaScript
if (navigator.msSaveBlob) { // IE10 / IE11
  if (!navigator.msSaveBlob(blob, filename)) {
    this.downloadUrl(url, filename);
  }
  return;
}
```
And then moving this block *after* it instead (and I don't think you need the `!URL` check any more, since we now using a `URL` polyfill).
```
if (pdfjsLib.PDFJS.disableCreateObjectURL) {
  // URL.createObjectURL is not supported.
  this.downloadUrl(url, filename);
  return;
}
```